stdlib/Markdown/src/GitHub/table.jl

```diff
@@ -142,11 +142,11 @@ function term(io::IO, md::Table, columns)
     padcells!(cells, md.align, len = ansi_length)
     for i = 1:length(cells)
         join(io, cells[i], " ")
-        println(io)
         if i == 1
-            join(io, ["–"^ansi_length(cells[i][j]) for j = 1:length(cells[1])], " ")
             println(io)
+            join(io, ["–"^ansi_length(cells[i][j]) for j = 1:length(cells[1])], " ")
```

Comment:
Would it be worth using `'-'` here as you've done in other places instead of `"-"`?

Reply:
Probably.   There is a lot of this idiom in the Markdown module, however, and my inclination would be to leave it to a separate PR.

Reply:
That would be fine, but I don't think 1-character strings are really so bad.

Reply:
Yes, they are a lot cheaper than a couple of versions ago.   And Markdown rendering is not generally performance-critical.